src/tests.rs

```diff
@@ -61,7 +61,10 @@ fn test_rpm_header_base(package: RPMPackage) -> Result<(), Box<dyn std::error::E
     );
     assert_eq!(package.metadata.get_build_time().unwrap(), 1540945151);
     assert_eq!(package.metadata.get_installed_size().unwrap(), 503853);
-    assert_eq!(package.metadata.get_payload_compressor().unwrap(), "xz");
+    assert_eq!(
+        package.metadata.get_payload_compressor().unwrap(),
+        CompressionType::Xz { level: 9 }
```

Comment:
No, the `CompressionType` should be just the type, unadorned by levels.   We're actually just making up a value here, I don't want to do that